test/TestConfig/resources/excludes/latest_exclude_SE90.txt

```diff
@@ -27,3 +27,6 @@ org.openj9.test.java.lang.Test_ClassLoader																				134754 generic-all
 org.openj9.test.java.lang.Test_Thread:test_getContextClassLoader														125908 generic-all
 org.openj9.test.vmArguments.VmArgumentTests:testCrNocr																	244 generic-all
 j9vm.test.softmx.SoftmxRemoteTest																						480 generic-all
+# Exclude this while investigating https://github.com/eclipse/openj9/issues/2047
+# Intermittent NullPointerException in TestAttachErrorHandling.deleteIpcDirectories
+org.openj9.test.attachAPI.TestAttachErrorHandling:test_startupShutdownRobustness
```

Comment:
missing issue number and platform. We can remove line `Exclude this while investigating https://github.com/eclipse/openj9/issues/2047`